combobox.py

```diff
@@ -0,0 +1,25 @@
+import sys
+from PyQt5.QtCore import *
+from PyQt5.QtGui import *
```

Comment:
Could you avoid importing all(`*`)? Also, supporting PyQt4 is a virtue

Reply:
I have fixed the imports and added PyQt4 support based on the implementation in labelImg.py script. I don't have PyQt4 on my environment so I couldn't test the support myself.

Reply:
@tzutalin Any chance that you can review this again? Thank you